src/unclutter.c

```diff
@@ -141,6 +144,19 @@ static void parse_args(int argc, char *argv[]) {
                     break;
                 } else if (OPT_NAME_IS("ignore-scrolling")) {
                     config.ignore_scrolling = true;
+                    break;
+                } else if (OPT_NAME_IS("ignore-buttons")) {
+                    value = parse_buttons_numbers(optarg, &config.ignore_buttons);
+                    if (value == ULONG_MAX) {
+                        ELOG("Invalid ignore-buttons value specified.");
+                        config.ignore_buttons.count = 0;
+                    } else {
+                        // shrink array size
+                        config.ignore_buttons.buttons = (unsigned int *)realloc(
```

Comment:
This will become unnecessary with my comments below about reallocating as we go.